ReactAndroid/src/main/java/com/facebook/react/uimanager/BaseViewManager.java

```diff
@@ -115,13 +115,13 @@ public void setScaleY(T view, float scaleY) {
   @Deprecated
   @ReactProp(name = PROP_TRANSLATE_X, defaultFloat = 1f)
```

Comment:
Seems like this was copy-paste kind of an issue from `setScale`. Also `default` should be `0f` here instead of `1f`. Do you mind updating `defaultFloat` here too to completely fix translate property related regression introduced by refactoring? Once it's done I'll be happy to merge your patch